source/common/config/grpc_mux_impl.h

```diff
@@ -83,12 +95,19 @@ class GrpcMuxImpl : public GrpcMux,
     bool pending_{};
     // Has this API been tracked in subscriptions_?
     bool subscribed_{};
+    // Detects when Envoy is making too many requests.
+    // Bucket contains 90 tokens max and refills at 5 tokens/sec.
+    TokenBucket limit_request{90, 5};
```

Comment:
Would constants make more sense than magic numbers with comments?

Reply:
I think eventually this might be configurable. It's fine to leave a TODO and keep it as is for now IMHO.

Reply:
OK cool